fitlins/interfaces/nistats.py

```diff
@@ -37,6 +43,9 @@ def _run_interface(self, runtime):
         import nibabel as nb
         from nistats import design_matrix as dm
         info = self.inputs.session_info
+        drop_missing = self.inputs.drop_missing
```

Comment:
`bool(Undefined) == False`.

```suggestion
        drop_missing = bool(self.inputs.drop_missing)
```